robots/flakefinder/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main_test
```

Comment:
This is the cause for the intellij error.
Normally is good practice to have the tests package `xxx_test`, because it will force you only to test the "public api" of a package. 
In this case though, everything belongs to the main package so it does not really make any difference and you can just have `package main` for the test file.
In this way, it will be able to access package private names. I think it's still better than having mixed exported names because you need only some of them in the test.

Reply:
Sounds good, but I will need to adjust the code at other places also.

Reply:
BTW thanks for the explanation!

Reply:
for the purpouse of this patch I think you can just revert options to be private and have all the fields exported in the same way.

Reply:
Done!